src/debug_sdfs_ed64.c

```diff
@@ -0,0 +1,590 @@
+
+/*********************************************************************
+ * FAT backend: Everdrive64
+ *********************************************************************/
+static volatile struct PI_regs_s * const PI_regs = (struct PI_regs_s *)0xa4600000;
+
+// Everdrive registers
+#define ED64_BASE_ADDRESS             0xBF808000
+#define ED64_SD_IO_BUFFER                 0x0200
+#define ED64_REGISTER_SD_BASE             0x0020
+#define ED64_REGISTER_SD_STATUS           0x0030
+
+// Everdrive config bits
+#define ED64_SD_CFG_BITLEN                0x000F
+#define ED64_SD_CFG_SPEED                 0x0010
+
+// Everdrive status bits
+#define ED64_SD_STATUS_BUSY               0x0080
+
+// If timed out, we always return 0xFF and set the start bit to 1 as well
+#define ED64_SD_ACMD41_TOUT_MS          1000
+#define ED64_SD_ACMD41_WAIT_MS            10
+#define ED64_SD_TIMEOUT                 0x80
+
+// Everdrive SD mode commands
+#define ED64_SD_CMD0                    0x40 // Go idle state
+#define ED64_SD_CMD2                    0x42 // All send CID
+#define ED64_SD_CMD3                    0x43 // Send relative addr
+#define ED64_SD_CMD6                    0x46
+#define ED64_SD_CMD7                    0x47 // Select/deselect card
+#define ED64_SD_CMD8                    0x48 // Send interface condition
+#define ED64_SD_CMD12                   0x4C // Stop transmission on multiple block read
+#define ED64_SD_CMD18                   0x52 // Read multiple block
+#define ED64_SD_CMD25                   0x59 // Write multiple block
+#define ED64_SD_CMD55                   0x77 // Application specific cmd
+#define ED64_SD_CMD41                   0x69
```

Comment:
I think I saw that the commands are really called "CMDnn" but do you think it's better to keep them like this for consistency? Because the code is really unreadable with these names, you don't understand what a command does without coming here for the comment.

Reply:
The issue is that they can mean different things when issued after a `CMD55` THen they become `ACMDnn` That's why I don't even have the comments here for `ED64_SD_CMD41` and `ED64_SD_CMD6` here. I think this is easier for when following them on the spec as well. I generally did the navigation in the spec :)